test/example.md

```diff
@@ -183,6 +183,12 @@ _Returns:_ [`<number>`][number] result of `fn` execution
 
 Function with function
 
+## incorrectDocsFunction(b)
+
+- `b`: [`<string>`][string]
+
+This function has incorrect parameters docs indentation
```

Comment:
This should be tested separately with a manual `exec` of a `metadoc` and `.on('warning'..)` listener, as it'd be strange to see warnings during a normal `npm test` run.